src/R/Components/Impl/ConnectionManager/Implementation/ConnectionManager.cs

```diff
@@ -64,6 +64,12 @@ internal class ConnectionManager : IConnectionManager {
             _userConnections = new ConcurrentDictionary<Uri, IConnection>(userConnections);
 
             UpdateRecentConnections();
+
+            _shell.Idle += OnIdle;
+        }
+
+        private void OnIdle(object sender, EventArgs e) {
```

Comment:
Well, `DTEEvents.OnStartupComplete` happens only after status bar container is created and it is on main thread. 
I've just tried to repro it and it worked as expected. Do I need your changes to repro the issue?

---
In reply to: [78463049](https://github.com/Microsoft/RTVS/pull/2296#discussion_r78463049) [](ancestors = 78463049)